tron/actioncommand.py

```diff
@@ -195,6 +201,15 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    @staticmethod
+    def to_json(state_data: dict) -> str:
+        return json.dumps(
+            {
+                "status_path": state_data["status_path"],
+                "exec_path": state_data["exec_path"],
+            }
+        )
```

Comment:
oh, i see - a lot of these have `state_data()` properties! i think that  that makes this less pressing since the calls will (i assume) look like `SomeClass.to_json(some_object.state_data)

but it might still be nicer to switch since for things that do implement state_data() this would look like
```suggestion
    def to_json(self) -> str:
        state_data = self.state_data
        
        return json.dumps(
            {
                "status_path": state_data["status_path"],
                "exec_path": state_data["exec_path"],
            }
        )
```


and for classes that don't have a `state_data` property we'd be in the scenario described above and would benefit from this no longer being a staticmethod :)

Reply:
(another benefit would also be avoiding any potential circular imports in the future since we won't need to import classes just to call to_json() :)

Reply:
oh, i see - when this gets called in tron/serialize/runstate/dynamodb_state_store.py we just have a key and a dict ;_;

Reply:
hmm, maybe this is better for a post-pickle-deletion refactor where we update the simplified code to pass around the actual objects rather than state_data dicts